_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -527,3 +527,17 @@ function jetpack_current_user_data() {
 
 	return $current_user_data;
 }
+
+/**
+ * Return the username of the Jetpack connection's owner.
+ *
+ * @since 7.8.0
+ *
+ * @return array
+ */
+function jetpack_connection_owner_name() {
```

Comment:
Noting that we clean things up a bit more in other places, like here:
https://github.com/Automattic/jetpack/blob/ea68b7afa0067d34a6182eaaa98f82a717627455/_inc/lib/debugger/class-jetpack-debug-data.php#L365-L384

I think we could do something similar here.

cc @kbrown9 Do you think this would make a good candidate for a new method somewhere in the Connection package? I could imagine others wanting to use something like this in the future.

Reply:
> Do you think this would make a good candidate for a new method somewhere in the Connection package? I could imagine others wanting to use something like this in the future.

Yes, I think it makes sense to put this new method in the Connection package. I think the [Manager class](https://github.com/Automattic/jetpack/blob/master/packages/connection/src/class-manager.php) is a good location.

@Aurorum - what do you think of putting this in the Manager class, with the method name `get_connection_owner_name()`? 

Then in `Jetpack_Core_Json_Api_Endpoints::get_user_connection_data()`, we would call `( new Connection_Manager() )->get_connection_owner_name() `.

Reply:
Ooh, I didn't know about that, thanks! It looks like information on the connection owner is also stored there.

https://github.com/Automattic/jetpack/blob/51a4f6caa2c1dc4909d7572698af8341c4576e21/packages/connection/src/class-manager.php#L632-L646

In which case, would something like the following be preferable or would it be better to have a separate `get_connection_owner_name()` there? 

`'connectionOwner' => ( new Connection_Manager() )->get_connection_owner()->data->display_name,`

Reply:
Your suggestion to use `( new Connection_Manager() )->get_connection_owner()->data->display_name` looks good to me!

Reply:
Thanks, I've updated the PR with that change! :)